src/component/ModalForm.jsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import Backdrop from "@mui/material/Backdrop";
+import Box from "@mui/material/Box";
+import Modal from "@mui/material/Modal";
+import Fade from "@mui/material/Fade";
+import Typography from "@mui/material/Typography";
+import { Button, Divider} from "@mui/material";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import * as Yup from "yup";
+
+const validationSchema = Yup.object().shape({
+  name: Yup.string()
+    .min(5, "Name must be at least 5 characters")
+    .max(30, "Name cannot exceed 30 characters")
+    .required("Name is required"),
+  email: Yup.string()
+    .email("Invalid email address")
+    .required("Email is required"),
+});
+
+const initialValues = {
+  name: "",
+  email: "",
+};
+const style = {
```

Comment:
As you are using MUI, why you are not using `makeStyle` & `createStyle`

Reply:
well, I will use it in the next task